pkg/controller/clustersync/clustersync_controller.go

```diff
@@ -924,16 +946,17 @@ func applyToTargetCluster(
 		return err
 	}
 
+	logger.Info("attempting to apply resource")
```

Comment:
Think we should put the UID/ResourceVersion in this one too? Esp the latter, I would expect it to change on apply.